src/web/components/SharingPermission/bulkAddPermissionsHelpers.ts

```diff
@@ -0,0 +1,57 @@
+import { formatStringsWithSeparator, getArticle } from '../../utils/textHelpers';
```

Comment:
Not sure how we feel about helper files, but this seemed like stuff that could be moved out of the component. Let me know what you think